src/json/test_data/volleyball-for-boys.js

```diff
@@ -4,22 +4,29 @@ export default {
     'http://iiif.io/api/presentation/3/context.json',
   ],
   type: 'Manifest',
-  id:
-    'https://dlib.indiana.edu/iiif-av/iiif-player-samples/volleybal-for-boys/manifest',
+  id: 'https://dlib.indiana.edu/iiif-av/iiif-player-samples/volleybal-for-boys/manifest',
```

Comment:
It looks like this id is missing the second `l` in volleyball.  I think it would be nice to fix this throughout the file if possible.